crates/eframe/src/native/run.rs

```diff
@@ -39,7 +39,7 @@ fn create_event_loop(native_options: &mut epi::NativeOptions) -> EventLoop<UserE
     let mut builder = create_event_loop_builder(native_options);
 
     crate::profile_scope!("EventLoopBuilder::build");
-    builder.build()
+    builder.build().unwrap()
```

Comment:
can we either change the api to return a `Result` or should we update corresponding public api docs to warn about a possible panic here?

Reply:
I'll change it to return errors